module.c

```diff
@@ -319,6 +308,22 @@ static PyObject *context_module(ContextData *self, PyObject *args) {
 	return context_eval_internal(self, args, JS_EVAL_TYPE_MODULE);
 }
 
+// _quickjs.Context.parse_json
+//
+// Evaluates a Python string as JSON and returns the result as a Python object. Will
+// return _quickjs.Object for complex types (other than e.g. str, int).
+static PyObject *context_parse_json(ContextData *self, PyObject *args) {
+	const char *data;
+	if (!PyArg_ParseTuple(args, "s", &data)) {
+		return NULL;
+	}
+	JSValue value;
+	Py_BEGIN_ALLOW_THREADS;
+	value = JS_ParseJSON(self->context, data, strlen(data), "data.json";
```

Comment:
missing ), and maybe pass `context_parse_json` or similar as the filename argument

Reply:
Done.